LICENSE

```diff
@@ -1,8 +1,9 @@
-Java Native Access project (JNA) is dual-licensed under 2 
-alternative Open Source/Free licenses: LGPL 2.1 or later and 
-Apache License 2.0. (starting with JNA version 4.0.0). 
+SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1
 
-You can freely decide which license you want to apply to 
+This library is licensed under the LGPL, version 2.1 or later,
```

Comment:
I would retain the library name here, people sharding JNA and merging META-INF will be bitten hard else.